### ISSUE TITLE
applications take very long to start

### ISSUE THREAD

**Issue by taukakao** [Vanilla-OS/desktop-image#126]
### Issue Description

After logging out and back in, applications launched through GNOME take a very long time to start.

### Steps to Reproduce

- (re-)boot system
- log out
- log in
- start app

### On what version of Vanilla OS this happens?

Unreleased

**Comment by mirkobrombin** [comment]
Just tested, same issue here

**Comment by Guztaver** [comment]
same here, apps take 2min just to start showing up. 
but, to me, just occurs when log out and log in.

I'm using Vanilla Orchid

**Comment by taukakao** [comment]
Maybe related:

When this happens and you're shutting down, xdg-document-portal.service fails to quit completely. The logs of xdg-document-portal say:
`fusermount3: failed to open /etc/mtab: No such file or directory`

**Comment by taukakao** [comment]
Yeah, seems like this is related to the broken symlink /etc/mtab.

**Comment by Guztaver** [comment]
> Yeah, seems like this is related to the broken symlink /etc/mtab.

in my pc, this symlink is just normal.
![image](https://github.com/Vanilla-OS/desktop-image/assets/39814943/6be216be-2773-4f19-87a0-f7ba86484f13)

**Comment by taukakao** [comment]
@Guztaver 
You're inside a container with a seperate /etc

The problem is that /etc/mtab of the host is reset on every boot. So changing it once doesn't help.
